src/pages/_app.tsx

```diff
@@ -42,7 +42,7 @@ function MyApp({ Component, pageProps }: AppProps) {
   return (
     <main className={`${GeistSans.variable} font-primary ${menloFont.variable}`}>
       <Head>
-        {/* <link
+        <link
```

Comment:
Would love to hear more context about this one. It's tricky. Ideally we would like to get the favicons from `getStaticProps` to get the favicons only on build time, but the delegate page is already using `getServerSideProps` which is called at every page request.